sdk/identity/azure-identity/azure/identity/_internal/interactive.py

```diff
@@ -176,15 +227,16 @@ def get_token(
             self._auth_record = _build_auth_record(result)
         except Exception as ex:  # pylint:disable=broad-except
             _LOGGER.warning(
-                "%s.get_token failed: %s",
+                "%s.%s failed: %s",
                 self.__class__.__name__,
+                base_method_name,
                 ex,
                 exc_info=_LOGGER.isEnabledFor(logging.DEBUG),
             )
             raise
 
-        _LOGGER.info("%s.get_token succeeded", self.__class__.__name__)
-        return AccessToken(result["access_token"], now + int(result["expires_in"]))
+        _LOGGER.info("%s.%s succeeded", self.__class__.__name__, base_method_name)
+        return AccessTokenInfo(result["access_token"], now + int(result["expires_in"]))
```

Comment:
Don't forget the token info. :)

Reply:
_request_token may return a pop token.

Reply:
Good catch! Updated to now propagate "token_type" if available (and also refresh_on here as well)